integration/kubernetes/init.sh

```diff
@@ -133,6 +133,7 @@ sed -i "s|CGROUP_DRIVER|${cgroup_driver}|" "${kubeadm_config_file}"
 trap 'sudo -E sh -c "rm -r "${kubeadm_config_file}""' EXIT
 
 if [ "${CI}" == true ] && [[ $(wc -l /proc/swaps | awk '{print $1}') -gt 1 ]]; then
+	grep -q zram /proc/swaps && echo "# zram swap disabled"  | sudo tee /etc/systemd/zram-generator.conf
```

Comment:
Nit: It would be safer to `tee -a` in case that file already exists.

Reply:
but then it won't disable swap no?

Reply:
I thought  if the file existed, swap was disabled? But if it already exists swap will already be disabled - the script doesn't check if the file exists or not. So if it does exist, it might have some other content so it would be safer to append to it.

Reply:
It's mentioned that in order to disable `you can override the default config file with an empty one`
By default this file doesn't exist, if you do create it and set some custom zram configurations it will use those, hence, appending a comment to an existing custom configuration won't help disabling zram

(I plan to disable zram in kernel cmdline or by deleting the pkg from the f35 ci images and also update the relevant documentation)

Reply:
@snir911 does Kubernetes fail if zram swap is on?